docs/design/design_v1.9.md

```diff
@@ -440,14 +490,21 @@ Please note that:
 - The temporary authentication resolve to a user member of  `system:bootstrappers:kubeadm:default-node-token` group which was granted access to CSR api during the `kubeadm init` process
 - The automatic CSR approval is managed by the csrapprover controller, according with configuration done the `kubeadm init` process
 
+### (optional) Write init kubelet configuration
+
+If kubeadm is invoked with `--feature-gates=DynamicKubeletConfig`: 
+
+1. Read the kubelet base configuration from the `kubelet-base-config-v1.9`  ConfigMap in the `kube-system` namespace, and write it to disk as kubelet init configuration file  `/var/lib/kubelet/config/init/kubelet`.
+2. As soon as kubelet starts, update current node configuration specifying that the the source for the node/kubelet configuration is the above ConfigMap.
```

Comment:
This is done _using the Node's own credential_ (`/etc/kubernetes/kubelet.conf`)
You could copy the text I proposed above for this